src/main/java/org/sunbird/workflow/service/impl/WorkflowServiceImpl.java

```diff
@@ -771,17 +764,20 @@ public Response getUserWf(String rootOrg, String org, String wid, SearchCriteria
 
 	@Override
 	public Response getUserWFApplicationFields(String rootOrg, String org, String wid, SearchCriteria criteria) {
-		List<String> updatedFieldValues = wfStatusRepo.findWfFieldsForUser(rootOrg, org, criteria.getServiceName(), criteria.getApplicationStatus(), wid);
+		List<Object[]> updatedFieldValues = wfStatusRepo.findWfFieldsForUser(rootOrg, org, criteria.getServiceName(), criteria.getApplicationStatus(), wid);
 		TypeReference<List<HashMap<String, Object>>> typeRef = new TypeReference<List<HashMap<String, Object>>>() {
 		};
-		Map<String,Object> toValuesMap = new HashMap<>();
-		for (String fields : updatedFieldValues) {
+		List<Map<String, Object>> result = new ArrayList<>();
+		for (Object[] fields : updatedFieldValues) {
 			if (!StringUtils.isEmpty(fields)) {
 				try {
-					List<HashMap<String, Object>> values = mapper.readValue(fields, typeRef);
+					List<HashMap<String, Object>> values = mapper.readValue((String)fields[0], typeRef);
 					for (HashMap<String, Object> wffieldReq : values) {
+						Map<String, Object> resultData = new LinkedHashMap<>();
 						HashMap<String, Object> toValueMap = (HashMap<String, Object>) wffieldReq.get("toValue");
-						toValuesMap.put(toValueMap.entrySet().iterator().next().getKey(),toValueMap.entrySet().iterator().next().getValue());
+						resultData.put("wfId", fields[1]);
+						resultData.put(toValueMap.entrySet().iterator().next().getKey(),toValueMap.entrySet().iterator().next().getValue());
+						result.add(resultData);
```

Comment:
in the catch block - do update the logger and use log.error(message, exception) method to log errors.

Reply:
This is done with additional changes.